### ISSUE TITLE
Support for saveBloodGlucose(), getCarbohydrates() and saveCarbohydrates()

### ISSUE THREAD

**Issue by snics** [lucaspbordignon/rn-apple-healthkit#72]
Hello,
Is it possible to add the following functions? I would like to insert as well as read out blood clucose values. Additionally, I would like to save as well as read out carbohydrate values (https://developer.apple.com/documentation/healthkit/hkquantitytypeidentifier/1615699-dietarycarbohydrates). Would it be possible for you to provide this?

**Comment by lucaspbordignon** [comment]
@snics Hey! It has been a while since you've added this request but as the repository is not being actively maintained now, I've created a new [NPM](https://www.npmjs.com/package/react-native-health) package and started to maintain and add new features there

The repo is [this](https://github.com/agencyenterprise/react-native-health) one and I've added these additions as feature requests in the project. Will keep you posted in case we have any update and also feel free to help in case this is still relevant for you!
